planning/behavior_velocity_planner/include/scene_module/speed_bump/manager.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Tier IV, Inc., Leo Drive Teknoloji A.Ş.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SCENE_MODULE__SPEED_BUMP__MANAGER_HPP_
+#define SCENE_MODULE__SPEED_BUMP__MANAGER_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+#include <scene_module/scene_module_interface.hpp>
+#include <scene_module/speed_bump/scene.hpp>
+
+#include <autoware_auto_planning_msgs/msg/path_with_lane_id.hpp>
+
+#include <functional>
+#include <memory>
+
+namespace behavior_velocity_planner
+{
+class SpeedBumpModuleManager : public SceneModuleManagerInterfaceWithRTC
```

Comment:
This module seems not to need perception information.
So, this module does not need to use RTC interface.
https://github.com/autowarefoundation/autoware.universe/issues/891

In such case, please inherit `SceneModuleManagerInterface` instead of `SceneModuleManagerInterfaceWithRTC`.
The example of this is implemented in following.
https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_velocity_planner/include/scene_module/stop_line/manager.hpp

Reply:
@rej55 

Thank you for the comment. I totally missed that part.